### PR TITLE
docs: extend idle connection timeout on ping tutorial

### PR DESCRIPTION
**Pull request by rvalle**
Cosmetic fix on tutorial. Avoids confusion. 

addresses #5175

### CODE REVIEW

**Review by retrohacker — APPROVED** [review]
LGTM :+1: 

Thank you for doing this

File: libp2p/src/tutorials/ping.rs [hunk 1]
```diff
@@ -216,7 +216,7 @@
 //!             libp2p::yamux::Config::default,
 //!         )?
 //!         .with_behaviour(|_| ping::Behaviour::default())?
-//!         .with_swarm_config(|cfg| cfg.with_idle_connection_timeout(Duration::from_secs(30))) // Allows us to observe pings for 30 seconds.
+//!         .with_swarm_config(|cfg| cfg.with_idle_connection_timeout(Duration::from_secs(u64::MAX))) // Allows us to observe pings for a long time.
```
Comment (jxs) [inline libp2p/src/tutorials/ping.rs]:
```suggestion
//!         .with_swarm_config(|cfg| cfg.with_idle_connection_timeout(Duration::from_secs(u64::MAX))) // Allows us to observe pings indefinitely.
```

File: libp2p/src/tutorials/ping.rs [hunk 2]
```diff
@@ -267,7 +267,7 @@
...
```
Comment (jxs) [inline libp2p/src/tutorials/ping.rs]:
```suggestion
//!         .with_swarm_config(|cfg| cfg.with_idle_connection_timeout(Duration::from_secs(u64::MAX))) // Allows us to observe pings indefinitely..
```

File: libp2p/src/tutorials/ping.rs [hunk 3]
```diff
@@ -312,7 +312,7 @@
...
```
Comment (jxs) [inline libp2p/src/tutorials/ping.rs]:
```suggestion
//!         .with_swarm_config(|cfg| cfg.with_idle_connection_timeout(Duration::from_secs(u64::MAX))) // Allows us to observe pings indefinitely.
```

**Review by jxs — APPROVED** [review]
LGTM

**Review by dariusc93 — APPROVED** [review]

**Review by jxs — APPROVED** [review]

**Pull request merged**